src/script/view_model/ContentViewModel.ts

```diff
@@ -68,16 +148,13 @@ export class ContentViewModel {
     };
   }
 
-  constructor(mainViewModel, repositories) {
-    this.switchContent = this.switchContent.bind(this);
-    this.switchPreviousContent = this.switchPreviousContent.bind(this);
-
+  constructor(mainViewModel: MainViewModel, repositories: Repositories) {
     this.elementId = 'center-column';
     this.mainViewModel = mainViewModel;
     this.conversationRepository = repositories.conversation;
     this.userRepository = repositories.user;
     this.logger = getLogger('ContentViewModel');
-    this.STATE = ContentViewModel.STATE;
+    this.State = ContentViewModel.STATE;
```

Comment:
Why not keep `STATE`?

Reply:
typescript says property STATE already exists

Reply:
Hmm okay